tests/jballerina-unit-test/src/test/resources/test-src/workers/sync-send.bal

```diff
@@ -629,6 +629,211 @@ public function testFailureForReceiveWithError() returns boolean {
     return x.w1 is E2 && x.w2 === true;
 }
 
+string appendCloneable = "";
+function testSimpleSyncSendWithCloneableExpression() {
```

Comment:
Is it correct to say sync send and have these tests in this file? We seem to have normal sends in processWithCloneableExpression?

Reply:
I will move these